app/models/carrier.rb

```diff
@@ -7,10 +7,9 @@ class Carrier < ApplicationRecord
   has_many :outbound_sip_trunks
   has_many :phone_numbers
   has_many :phone_calls
-
-  has_one :application,
-          class_name: "Doorkeeper::Application",
-          as: :owner
+  has_many :events
+  has_one :oauth_application, as: :owner
+  has_one :webhook_endpoint, through: :oauth_application
```

Comment:
Why don't we have webhook_endpoint has association with carrier directly?

I'm not 100% sure, but as I remember we want to support multiple applications for one entity in another system, but for this app, but in here carrier has only one webhook endpoint.